app_data/vocabularies/resource_types.yaml

```diff
@@ -656,7 +656,7 @@
     openaire_type: dataset
```

Comment:
For this vocabulary the datacite_general attribute is Model, but we display it as an subtype of an image. To me it looks it should not be a subtype of image as if we follow the [datacite guidelines](https://support.datacite.org/docs/what-are-the-resource-types-for-datacite-dois) `model` is available, same for the openaire guidelines (https://guidelines.openaire.eu/en/latest/data/field_resourcetype.html), should it be a `dataset` or `model`?. Alternatively if we want to keep it as subtype, should we change it to:
```diff
-    datacite_general: Model
-    datacite_type: ''
+    datacite_general: Image
+    datacite_type: Model
```